src/components/paint_format_button/paint_format_button.ts

```diff
@@ -0,0 +1,46 @@
+import { Component } from "@odoo/owl";
+import { SpreadsheetChildEnv } from "../../types";
+import { css } from "../helpers";
+
+css/* scss */ `
+  .o-menu-item-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 2px;
+    border-radius: 2px;
+    min-width: 20px;
+  }
```

Comment:
Don't redefine a class that is already defined elsewhere. The code is already messy ATM, we use this class in the topbar when it's defined in acton button... We should probably move the class in spreadsheet.ts

Reply:
Or just pass the class as props